src/enzax/steady_state.py

```diff
@@ -5,12 +5,83 @@
 """  # noqa: E501
 
 import diffrax
+import jax
 import equinox as eqx
 import lineax as lx
+import optimistix as optx
+
 from jaxtyping import Array, Float, PyTree
+from jax import numpy as jnp
+
+from enzax.kinetic_model import KineticModel, RateEquationModel
+
+
+@eqx.filter_jit()
+def dC_dt_sqrd(
+    model: RateEquationModel,
+    x: Float[Array, " n_balanced"],
+    conc: Float[Array, " n_met"]
+) -> Float:
+    S = model.structure.S
+    dG = (S.T @ model.parameters.dgf + 2.4788191*S.T@jnp.log(conc))
+    return sum(jnp.square(model.dcdt(0, x)))
```

Comment:
maybe either `jnp.square(model.dcdt(0, x)).sum()` or `jnp.sum(jnp.square(model.dcdt(0, x)))` would be better here - I'm not sure exactly what happens when you use the standard `sum`